### PR TITLE
Added post_init

### PR DESCRIPTION
**Pull request by trag1c**

### CODE REVIEW

**Comment by trag1c** [comment]
@Lunarmagpie review pretty please 🥺

**Review by MithicSpirit — CHANGES_REQUESTED** [review]

File: src/library-lagoon/post_init.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Task:
+    name: str
+    priority: int
+
+    def __post_init__(self) -> None:
+        self.priority = min(5, max(1, self.priority))  # capping priority to [1, 5]
```
[the remark this answers is not in the document]
Comment (trag1c) [reply src/library-lagoon/post_init.py:10]: Good point, thanks

**Review by Lunarmagpie — APPROVED** [review]
it looks pretty good, do what mithic said though

**Comment by trag1c** [comment]
Thanks guys

**Pull request merged**